documentation/builders/update.md

```diff
@@ -12,6 +12,98 @@ Restore your old files after the new installation was successful and check if ne
 $ diff shared/settings/jukebox.yaml resources/default-settings/jukebox.default.yaml
 ```
 
+## Manually upgrade to the latest version
+
+> [!CAUTION]
+> For optimal system updates, it is strongly recommended to utilize the upgrade feature when transitioning to the next version. Manual updates may necessitate specific migration steps and, if overlooked, could result in system failure.
+
+If you only want to update a few recent commits, this following explanation outlines the steps to do so
+
+Typically, 4 steps need to be considered
+
+1. Backup Local Changes (Optional)
+1. Pull the latest version from Github
+1. Replace the Web App with the most recent build
+1. Optional: Update the config files
+
+### Fetch the most recent version from Github
+
+First, SSH into your Phoniebox.
+
+```bash
+cd ~/RPi-Jukebox-RFID/
+```
+
+Second, get the latest version from Github. Depending on your proficiency with Git, you can also checkout a specific branch or version.
+Be aware, in case you have made changes to the software, stash them to keep them safe.
+
+1. Backup Local Changes (Optional):
+    - Stash your local changes:
+
+        ```bash
+        git stash push -m "Backup before pull"
+        ```
+
+    - Create a Backup Branch:
+
+        ```bash
+        git branch backup-before-pull
+        ```
+
+1. Pull Latest Changes:
+
+   ```bash
+   git pull
+   ```
+
+1. Update Web App:
+    1. Backup the current webapp build
```

Comment:
This should be done in compination with creating the backup branch, so move this to "Backup Local Changes"?

Reply:
The above step is optional: I am expecting users that have local changes to know that they have to backup them. The above is more a reminder rather than a necessity.
Backing up the Web App is slightly different. I want to be able to undo the change in case something went wrong.